### ISSUE TITLE
[BUG] Experiencing quick temporary overflows (scrollbar appears and disappears) when switching anchors for a global tooltip

### ISSUE THREAD

**Issue by PabloMessina** [ReactTooltip/react-tooltip#1035]
**Describe the bug**
I have a single global tooltip, defined as follows:

```Javascript
function App() {
   // ...
  // ...
    return (
        <div className={styles.app}>
            <div className={styles.header}>
                <h2>Chest X-ray Image and Report Annotation App</h2>
            </div>
            <div className={styles.content}>
                <MainReportView />
            </div>
            <div style={tooltip_div_style}>
                <Tooltip id="my-tooltip" />
            </div>
        </div>
    );
}
```

And I use it from different components:

```Javascript
function FeedbackForLabel({ labelName, labelIndex, reportFieldName, className }) {
    const for_gt_label = labelName !== undefined;
    const for_custom_label = labelIndex !== undefined;
    const for_report = reportFieldName !== undefined;
    if (for_gt_label + for_custom_label + for_report !== 1) {
        throw new Error("Exactly one of labelName, labelIndex, reportFieldName must be defined");
    }
    const report_data = store.get('report_data');
    let feedback;
    // let tooltip_id;
    if (for_gt_label) {
        feedback = report_data.get_feedback_for_gt_label_annotations(labelName);
        // tooltip_id = `tooltip-${labelName}`;
    } else if (for_custom_label) {
        feedback = report_data.get_feedback_for_custom_label_annotations(labelIndex);
        // tooltip_id = `tooltip-${labelIndex}`;
    } else if (for_report) {
        feedback = report_data.get_feedback_for_report_annotation(reportFieldName);
        // tooltip_id = `tooltip-${reportFieldName}`;
    }
    if (feedback.done) {
        return (<div className={className}>
                    <span className={styles['complete']}>Annotation complete &#x2713;</span>
                </div>);
    } else if (feedback.hasOwnProperty('messages') && feedback.messages.length > 0) {
        return (<div className={className} data-tooltip-id="my-tooltip"
            data-tooltip-html={ReactDOMServer.renderToStaticMarkup(
            <div style={tooltip_div_style}>
                <ul>{feedback.messages.map((message, index) => {
                        return (<li key={index}>{message}</li>);
                    })
                }</ul>
            </div>
            )}
        >
            <span className={styles['incomplete']}>Annotation incomplete &#x2717;</span>
        </div>);
    } else {
        return (<div className={className}>
            <span className={styles['incomplete']}>Annotation incomplete &#x2717;</span>
        </div>);
    }
}
```

```Javascript
function RoundQuestionButton({ link, style, tooltip_message }) {
    if (tooltip_message === undefined) {
        return (
            <a href={link} title={link} target="_blank" rel="noopener noreferrer" style={style}
            >
                <button style={button_style}>?</button>
            </a>
        );
    } else {
        return (
            <a href={link} title={link} target="_blank" rel="noopener noreferrer" style={style}
                data-tooltip-id="my-tooltip" data-tooltip-html={ReactDOMServer.renderToStaticMarkup(
                    <div style={tooltip_div_style}>{tooltip_message}</div>
                )}
            >
                <button style={button_style}>?</button>
            </a>
        );
    }
}
```
When I move my mouse from one component to the other and vice-versa, I can sometimes observe a scrollbar appearing and disappearing, suggesting that the tooltip is probably getting rendered and injected into the DOM in a way that momentarily takes up space and causes the surrounding container to overflow. If I add "overflow: hidden" to the div containing the tooltip, the tooltip won't show up at all, so that doesn't fix the problem.

**Version of Package**
v5.13.1

**To Reproduce**
Basically just try to create a setting like the one I described above. I'm including a GIF so that you can see the bug in action.

**Expected behavior**
There shouldn't be any spontaneous annoying overflows.

**Screenshots**
![tinder-like](https://github.com/ReactTooltip/react-tooltip/assets/6187545/bedd3039-fe58-494f-8f35-3efbde418754)

**Desktop (please complete the following information if possible or delete this section):**
 - OS: Windows 10
 - Browser: Chrome
 - Version 112.0.5615.138 (Official Build) (64-bit)
 - Frameworks: React^18.2.0

**Comment by gabrieljablonski** [comment]
Please try `positionStrategy="fixed"` and let us know if that makes any difference.

```tsx
<Tooltip positionStrategy="fixed" />
```

**Comment by PabloMessina** [comment]
Wow, that actually seems to have fixed the problem. What is the trick behind positionStrategy="fixed"?

Thank you very much for the super quick response!

**Comment by gabrieljablonski** [comment]
It is simply a passthrough to the `floating-ui` `computePosition()` `strategy` arg. More info here https://floating-ui.com/docs/computeposition#strategy

In summary, it sets the CSS `position` attribute. `absolute` is usually more efficient in terms of computation done by the browser, and `fixed` helps in situations involving scrolling (such as this one).

Glad we could help.
